torntools/scripts/content/loader/ttAttackLoader.js

```diff
@@ -5,7 +5,7 @@ requireDatabase().then(() => {
 });
 
 function displayWarning() {
-	if (userdata.energy.current > userdata.energy.maximum) {
+	if (userdata.energy.current > userdata.energy.maximum && userdata.chain.current > 0) {
```

Comment:
It should probably be 10 I guess, otherwise someone hitting on their own would make it that you won't get the notice.

Reply:
Done.